modules/nf-core/circularmapper/realignsamfile/main.nf

```diff
@@ -0,0 +1,52 @@
+process CIRCULARMAPPER_REALIGNSAMFILE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/circularmapper:1.93.5--h4a94de4_1':
+        'biocontainers/circularmapper:1.93.5--h4a94de4_1' }"
+
+    input:
+    tuple val(meta), path(bam)
+    tuple val(meta2), path(fasta)
+    val(elong)
+
+    output:
+    tuple val(meta), path("*_realigned.bam")    , emit: bam
+    path "versions.yml"                         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}" // Module actually has a hard-coded output of {input}_realigned.bam
```

Comment:
Can we mv the output file to include the prefix then.

Reply:
Added!